backend/homework/8/src/main/java/com/example/demo/repository/InventoryStore.java

```
@@ -0,0 +1,87 @@
+package com.example.demo.repository;
+
+import com.example.demo.logger.MyLogger;
+import com.example.demo.model.Vehicle;
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+@Data
+@Repository
+public class InventoryStore {
+    @Autowired
+   private List<Vehicle> vehicleList = new ArrayList<>();
+   public void addVehicle(Vehicle v)
+   {
+       vehicleList.add(v);
+   }
+   public Vehicle getVehicleByID(String id)
+   {
+    for(var vehicle : vehicleList)
+    {
+       if (vehicle.getId().equals(id))
+       {
+           return vehicle;
+       }
+
+
+    }
+       return null;
```

lets not return `null` rather throw an exception.